core/embed/rust/src/ui/layout_eckhart/fonts/font_ttsatoshi_extralight_72.rs

```diff
@@ -395,6 +395,105 @@ const Font_TTSatoshi_ExtraLight_72: [&[u8]; 95] = [
     &Font_TTSatoshi_ExtraLight_72_glyph_126,
 ];
 
+/// Array of references for 'TTSatoshi_ExtraLight_72' numeral-only ASCII glyphs
```

Comment:
this is not autogenerated right? in which case, we should at least add a note here, but better would be to add an option to gen_font to generate num-only variant too - so that we can reliably reproduce the file from vector font source.